src/pages/FactPage.tsx

```diff
@@ -0,0 +1,20 @@
+import "./FactPage.scss";
+import { getFacts } from "../data/factData";
+import FactCard from "../components/FactCard";
+
+const FactPage = () => {
+  return (
+    <div className="facts-wrapper">
+      <h1 className="page-header">Fun Facts</h1>
+
+      {getFacts().map((section) => (
+        <div>
+          <h2 className="section-header">{section.sectionTitle}</h2>
+          <FactCard section={section} />
+        </div>
+      ))}
```

Comment:
This is a list of things, so you should use (and style) a more semantic element. For example:
```suggestion
      <ul>
        {getFacts().map((section) => (
          <li>
            <h2 className="section-header">{section.sectionTitle}</h2>
            <FactCard section={section} />
          </li>
        ))}
      </ul>
```

Reply:
amended